pkg/piecestore/psserver/server_test.go

```diff
@@ -260,7 +252,7 @@ func TestRetrieve(t *testing.T) {
 					}),
 				}
 
-				s, err := cryptopasta.Sign(ba.Data, TS.k.(*ecdsa.PrivateKey))
+				s, err := cryptopasta.Sign(ba.Data, upID.Key.(*ecdsa.PrivateKey))
 				assert.NoError(err)
 				ba.Signature = s
```

Comment:
`ba.PubKey` is missing. How can this test case work if it send invalid bwagreements? Maybe the storage node is not validating them?

Reply:
given the information you've presented, I'd say the storage node is not validating them.  now I need to figure out why.

Reply:
The complete testset was disabled. I enabled them and was able to fix them.